src/main/java/bhs/devilbotz/subsystems/DriveTrain.java

```diff
@@ -659,4 +674,52 @@ public Command stopCommand() {
         },
         this);
   }
+
+  // =========================================================================
+  // photon vision sim
+
+  private void initVisionSim() {
+    // double CAMERA_PITCH_RADIANS = 1.0;
+    // double CAMERA_HEIGHT_METERS = 1.0;
+    // double TARGET_HEIGHT_METERS = 1.0;
+
+    // See
+    // https://firstfrc.blob.core.windows.net/frc2020/PlayingField/2020FieldDrawing-SeasonSpecific.pdf
+    // page 208
+    double targetWidth = Units.inchesToMeters(41.30) - Units.inchesToMeters(6.70); // meters
+    // See
+    // https://firstfrc.blob.core.windows.net/frc2020/PlayingField/2020FieldDrawing-SeasonSpecific.pdf
+    // page 197
```

Comment:
There is built in methods for defining the 2023 field. No need to specify specific dimensions. Also I would use april tags for localiation